evaluators/rank/NdcgEvaluator/tests/test_ndcg_evaluator.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+
+from .. import NDCGEvaluator
+
+@pytest.fixture(scope='function')
+def evaluator():
+    return NDCGEvaluator(eval_at=3)
+
+@pytest.fixture(scope='function')
+def desired():
+    return [.8, .4, .1, 0]
+
+@pytest.mark.parametrize('actual, use_traditional_formula, expected',
+    [
+        ([.4, .1, .8], True, 0.795),
```

Comment:
Missing lots of edge cases, what if one of the arrays is of length 0? What if eval at is 0? What if eval at is larger than any of the arrays? Desired should also be parametrized and not a fixture.
Test with different lengths of actual and desired